packages/api/src/mikro-orm/migrations/Migration20241119101706.ts

```diff
@@ -0,0 +1,7 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20241119101706 extends Migration {
+    async up(): Promise<void> {
+        this.addSql('alter table "BrevoConfig" add column "folderId" int;');
```

Comment:
Is folder id required or optional - in the db it is required but in the entity it is nullable?

Reply:
I set it to required, I guess that makes more sense, as we set it to 1 anyway: c48c00f7a092a0722cf5db4ab86ef160d2a4066f